### DISCUSSION TITLE
argo app stuck in deleting

### DISCUSSION THREAD

**Discussion by DienasV** [argoproj/argo-cd#20380] [Q&A]
level=info msg="Unable to delete application resources: application 'jetty-emvibexp' in namespace 'axioma' is not permitted to use project 'axioma'" application=jetty-emvibexp dest-namespace=axioma dest-server= reason=StatusRefreshed type=Warning
![image](https://github.com/user-attachments/assets/8fcf4f67-d941-455f-a703-2dcf9bd510d1)

**Comment by lesomnus** [comment]
Try to remove `annotation.filnalizers` field at Application
